Cargo.toml

```diff
@@ -4,11 +4,13 @@ members = [
     "fortanix-vme/eif-tools",
     "fortanix-vme/fortanix-vme-abi",
     "fortanix-vme/fortanix-vme-runner",
+    "fortanix-vme/nsm",
```

Comment:
Probably the nitro-enclaves-specific stuff shouldn't be in fortanix-vme.

Reply:
Yes I wanted to create a separate PR just for moving things. The `eif-tools` also shouldn't be directly in `fortanix-vme`